Tools/GenerateWebsiteData.ps1

```diff
@@ -77,6 +79,58 @@ function GetLinks ($docObj) {
     return $null
 }
 
+# Generate a PSObject that will be used to populate the cmdlet JSON file based on $parameter.
```

Comment:
Minor nit: it would be nice to know the type of `$parameter`. e.g. "based on $parameter (WidgetSproketFactory)."